repository.go

```diff
@@ -38,7 +38,7 @@ var ErrAggregateNotFound = errors.New("aggregate not found")
 
 // Repository is the returned instance from the factory function
 type Repository struct {
-	*EventStream
+	EventStream *EventStream
```

Comment:
@lunjon By making the EventStream a property on itself on the repository instead of an embedded. My feeling is that is possible to name the subscription methods with more specific names. Removing the pre Subscriber part of the name.

Reply:
I think this makes sense. Go with it!